src/collar/oc_folders.lsl

```diff
@@ -383,22 +410,23 @@ state active
             Dialog(g_kMenuUser, sPrompt, lButtons, ["+ Add Items", "- Rem Items", setor((g_sPath == ""), UPMENU, "^ UP")]+lLockOption, 0, g_iMenuUser, "FolderBrowser~");
         } else if(iChan == g_iFindChn)
         {
-            if(g_iCmdMode & F_RECURSIVE){
-                if(g_iCmdMode & F_ADD){
-                    llOwnerSay("@attachallover:"+sMsg+"=force");
-                } else if(g_iCmdMode & F_WEAR){
-                    llOwnerSay("@attachall:"+sMsg+"=force");
-                }else if(g_iCmdMode & F_REMOVE){
-                    llOwnerSay("@detachall:"+sMsg+"=force");
-                }
-            } else {
-                if(g_iCmdMode & F_ADD){
-                    llOwnerSay("@attachover:"+sMsg+"=force");
-                }else if(g_iCmdMode & F_WEAR){
-                    llOwnerSay("@attach:"+sMsg+"=force");
-                }else if(g_iCmdMode & F_REMOVE){
-                    llOwnerSay("@detach:"+sMsg+"=force");
+            list lParam = llParseStringKeepNulls(sMsg, [","],[]);
+            if(llGetListLength(lParam) > 1)
+            {
+                integer x = 0;
+                integer end = llGetListLength(lParam);
+
+                string sAppend = "";
+                for(x=0;x<end;x++)
+                {
+                    sAppend += "\n"+llList2String(lParam,x);
                 }
+                Dialog(kID, "There were multiple results found, please select one.\nOptions: \n\n"+sAppend, lParam, [], 0, g_iMenuUser, "FolderSearch");
+            } else if(llGetListLength(lParam) == 1){
+                ApplyFolderActions(sMsg);
+            } else if(llGetListLength(lParam) == 0)
+            {
+                llMessageLinked(LINK_SET, NOTIFY, "0No results were found", g_kMenuUser);

```

Comment:
Is "0No" a typo or should it be "No"?

Reply:
The 0/1 in front of the text determines if the wearer will be informed as well. This type of notification switching was implemented in 2015 or even earlier...